### DISCUSSION TITLE
Rustdesk is not able to work with IPv6

### DISCUSSION THREAD

**Discussion by iii0s** [rustdesk/rustdesk#5258] [General]
Hi;

It is a fantastic programme, however, using it with IPv6 is not stable and does not work reliably.

Whenever I connect using IPv6 I get "Not ready. Please check your connection". Still with this error, I can connect to any client, which shows "Ready.", Which means that that client is connected using IPv4. The connection from the client, which shows "Ready." to the client showing "Not ready. Please check your connection." does not work.

If I specify the IPv4 of my server in /etc/hosts on the client, the connection goes on smoothly.

I tested on different Linux distros, with different releases as well as on Windows 11.

Can anyone confirm this behaviour, please?

Thank you in advance.

**Comment by xlionjuan** [comment]
On Linux, macOS
run
`dig rustdesk.example.com AAAA` to ensure your result has no AAAA(IPv6) records

**Comment by iii0s** [comment]
Thank you xlionjuan. I have already done that. Problem solved, even though IPv6 is wished. :)
